README.md

```diff
@@ -55,6 +55,24 @@ jobs:
         args: analyse src/
 ```
 
+_to skip checking the platfrom requirements:_
+```diff
+on: [push, pull_request]
+name: Test
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: PHPStan
+      uses: docker://oskarstark/phpstan-ga
++     env:
++        CHECK_PLATFORM_REQUIREMENTS: false
```

Comment:
I suggest to reverse the logic. 

Name it `IGNORE_PLATFORM_REQUIREMENTS` instead. That will be the same as the composer flag

Reply:
I know, but it depends on the default, the default for `NO_DEV` is false, so you set it to true, the default is to *not* ignore platform reqs which sounds weird, that `IGNORE_PLATFORM_REQS: false` will actually ignore the platform reqs.
I did this in the first try.

Reply:
Hm.. Yeah. Okey. 

Yeah, that is fair. I didnt consider the other flag.

Reply:
Fully agree with @OskarStark here, I always try to avoid flags with "negative ideas", such as `DISABLE_XXX`.